packages/nutui-solid/src/components/cell/demos/icon.tsx

```diff
@@ -1,13 +1,12 @@
 import { Cell } from 'nutui-solid'
+import { User } from '@nutui/icons-solid'
```

Comment:
这个demo有点问题，cell的demo跟官网有点不同步
```suggestion
import { User } from '@nutui/icons-solid'
```